charts/netdata/README.md

```diff
@@ -148,6 +148,9 @@ The following table lists the configurable parameters of the netdata chart and t
 | `sysctlImage.tag`                        | sysctlImage Init container tag                                                                                                                         | `latest`                                                                  |
 | `sysctlImage.pullPolicy`                 | sysctlImage Init container pull policy                                                                                                                 | `Always`                                                                  |
 | `sysctlImage.resources`                  | sysctlImage Init container CPU/Memory resource requests/limits                                                                                         | {}                                                                        |
+| `wgetImage.repository`                   | wgetImage Init container name                                                                                                                        | `alpine`                                                                  |
```

Comment:
```suggestion
| `wgetImage.repository`                   | wgetImage Init container name                                                                                                                        | `netdata/wget`                                                                  |
```

Reply:
I switched to "alpine" because this is already used by sysctlImage and "alpine" image is correctly tagged with version number, so much easier to clone.

Reply:
we use custom image because we need gnu wget

https://github.com/netdata/helmchart/issues/92

Reply:
We can release tagged version of https://github.com/netdata/docker-wget